src/gpgpu-sim/gpu-cache.h

```diff
@@ -1191,21 +1191,21 @@ class cache_stats {
   void clear();
   // Clear AerialVision cache stats after each window
   void clear_pw();
-  void inc_stats(int access_type, int access_outcome);
+  void inc_stats(int access_type, int access_outcome, unsigned long long streamID=0);
   // Increment AerialVision cache stats
-  void inc_stats_pw(int access_type, int access_outcome);
-  void inc_fail_stats(int access_type, int fail_outcome);
+  void inc_stats_pw(int access_type, int access_outcome, unsigned long long streamID=0);
+  void inc_fail_stats(int access_type, int fail_outcome, unsigned long long streamID=0);
```

Comment:
Each kernel has a streamID. So a default value is not needed. And if people use the function without Stream ID the stats could be wrong. Let's take the default value out and always require a streamID

Reply:
Tim can correct me if I'm wrong, but in applications that have no streams, their stream will be the NULL stream (stream ID = 0).  So I think the default value will work if they use the function without stream ID in this case.

Reply:
Addressed in commit e8cc02d